tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1,12 +1,15 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring/
+# pylint: disable=missing-function-docstring, missing-module-docstring, unidiomatic-typecheck/
+import sys
 import pytest
-from numpy.random import rand, randint
+from numpy.random import rand, randint, uniform
 from numpy import isclose
 
 from pyccel.decorators import types
 from pyccel.epyccel import epyccel
 from conftest import *
```

Comment:
This import should no longer be required. It does not exist in the master branch. You probably need to merge the master branch into this branch (ideally, please do this in about 20 minutes once PR 498 has gone through)